stingray/deadtime/fad.py

```diff
@@ -0,0 +1,189 @@
+import warnings
+import numpy as np
+import scipy
+import matplotlib.pyplot as plt
+
+from scipy.ndimage.filters import gaussian_filter1d
+from scipy.interpolate import UnivariateSpline
+from astropy import log
+from astropy.table import Table
+
+from ..gti import cross_two_gtis, bin_intervals_from_gtis
+
+
+def _get_fourier_intv(lc, start_ind, end_ind):
+    time = lc.time[start_ind:end_ind]
+    counts = lc.counts[start_ind:end_ind]
+
+    fourier = scipy.fftpack.fft(counts)
+
+    freq = scipy.fftpack.fftfreq(len(time), lc.dt)
+    good = freq > 0
```

Comment:
Having the negative frequencies, if I'm not mistaken, is useful when we are interested in the complex quantities. Otherwise, the PDS and the other real-defined periodograms are just the same in the positive and negative frequencies. Am I missing something?

Reply:
I don't know, though I'm sure you're right. @abigailStev is the person to ask here. :) 
I just want to be sure we don't accidentally loose out on some functionality.